apps/web/src/App.tsx

```diff
@@ -38,9 +38,31 @@ import { BrandPage } from './pages/brand/BrandPage';
 import { SegmentProvider } from './store/segment.context';
 import LogRocket from 'logrocket';
 import setupLogRocketReact from 'logrocket-react';
+import packageJson from '../package.json';
```

Comment:
Did you had a chance to check if in some case this is affecting the build/dist folder structure of the web?

Reply:
What do you mean?

Reply:
@Cliftonz his suspicion was that the `package.json` won't be available after the build... but I've checked it and actually the `package.json` file is bundled to the app as the JSON string and then parsed... it just adds ~5.14kb to the bundled main JS chunk...

Reply:
I don't think so

Reply:
@scopsy This should be good to merge then